product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -91,3 +91,34 @@ Organization owners can assign users organization-level roles to complete certai
 5.  Select **Submit**.
 
 See [Adding a user to a project](/biganimal/release/administering_cluster/projects.mdx) for information on adding users to projects.
+
+## Machine users
+
+A machine user is used to drive non-GUI access to BigAnimal by user customized automation script, BigAnimal CLI commands, Terraform manifest, BigAnimal's first party binaries that runs from your environment.
+
+- Any user with the organization's owner role can create, update and delete a machine user.
+- The organization's owner user can be a normal user authenticated by IDP via OAuth2 flow or can be a machine user.
+- An organization owner can manage the Access key for the machine user but not for the normal users.
+- The organization owner and project owner can assign and unassign the roles to any machine user in the same organization.
+- Machine user is treated the same as the normal user by the BigAnimal AuthZ system on permission check.
+- The machine user is associated to one organization only and can't switch to another organization.
+- The machine user can't be invited.
+- The only way to authenticate and authorize a machine user is Access key.
+
+### Add machine user
+
+Only the organization's user having owner role can add a machine user.
+
+To add a machine user:
+
+1. Go to **User Management** from the menu next to your organization name in the top right of the portal.
+1. Select the **Add New User**.
+1. Select the **Machine User** as the user type.
+1. Provide the **Username** and **Email ID**.
+1. Optionally, select the **Create Access Key** check box.
```

Comment:
maybe we should explain why they would select the check box vs leave it unselected?